coverage/config.py

```diff
@@ -359,6 +360,7 @@ def copy(self):
         ('debug', 'run:debug', 'list'),
         ('disable_warnings', 'run:disable_warnings', 'list'),
         ('dynamic_context', 'run:dynamic_context'),
+        ('include_namespace_packages', 'run:include_namespace_packages', 'boolean'),
```

Comment:
Thanks.  Though I think this should be `'report:include_namespace_packages'`, since it affects how files are found during reporting, not during running.

Reply:
renamed